vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageLocationItem.kt

```diff
@@ -46,20 +53,32 @@ abstract class MessageLocationItem : AbsMessageItem<MessageLocationItem.Holder>(
         GlideApp.with(holder.staticMapImageView)
                 .load(location)
                 .apply(RequestOptions.centerCropTransform())
-                .into(holder.staticMapImageView)
+                .listener(object : RequestListener<Drawable> {
+                    override fun onLoadFailed(e: GlideException?, model: Any?, target: Target<Drawable>?, isFirstResource: Boolean): Boolean {
+                        holder.staticMapPinImageView.setImageResource(R.drawable.ic_location_pin_failed)
```

Comment:
This is maybe a design decision, but when the map fail to load, we could still draw the user avatar if available, no?

Reply:
CC @gaelledel

Reply:
Yes it is a design decision and it seems like a bug/ugly to use user avatar.

Reply:
I agree, thanks!